ts/screens/modal/IdentificationModal.tsx

```diff
@@ -502,6 +550,19 @@ class IdentificationModal extends React.PureComponent<Props, State> {
             {renderIdentificationByBiometryState(identificationByBiometryState)}
 
             <View spacer={true} large={true} />
+            {!isValidatingTask && (
+              <View style={styles.bottomContainer}>
+                <Text onPress={this.onLogout} alignCenter underlined white bold>
```

Comment:
Instead of using `Text` component we may use the Link from our Typography collection:
```typescript
                <Link onPress={this.onLogout} weight={"Bold"} color={"white"}>
                  {fromNullable(this.props.profileName).fold(
                    I18n.t("identification.logout"),
                    pN =>
                      I18n.t("identification.logoutProfileName", {
                        profileName: pN
                      })
                  )}
                </Link>
```

We should add the `white` color option and `Bold` weight to `Link` component, @fabriziofff @thisisjp do you think we can procede with these two minor improvements?

Reply:
@CrisTofani @fabriziofff sure—FYI we're working on Figma to release a new version of the UI Library, with a simplified approach that should help you (colors are independent styles, they're not "embedded" in the text styles)

Reply:
LGFM, we had agreed long ago to separate color from stylehttps://www.pivotaltracker.com/story/show/175577377, but there was never time to implement it. Maybe the new style library could be an opportunity to review them!